requirements.txt

```diff
@@ -1,13 +1,12 @@
-numpy==1.14.3
-torch>=0.4.1
-librosa==0.5.1
-Unidecode==0.4.20
-tensorboard
+scipy >=0.19.0
+torch >= 0.4.1
+librosa==0.6.2
+unidecode==0.4.20
 tensorboardX
 matplotlib==2.0.2
 Pillow
 flask
-scipy==0.19.0
-lws
+# lws
 tqdm
-git+git://github.com/bootphon/phonemizer@master
+soundfile
```

Comment:
Do we use soundfile? I remember we use it before for the server but I guess, I replaced it with librosa or scipy recently.

Reply:
I added it here because it was being installed separately in the Dockerfile. The comment said it was needed for the server. If that's not the case any more then we should drop it!